src/components/ChatLog.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog =(props) => {
+  
+  // const forLike = (value) => {alert(value)}
+
+  // console.log(props)
+  // console.log(props.allMessages)
```

Comment:
Reminder that we want to clean up print statements for development & debugging before opening PRs so they don't end up in our final codebase.